WorkflowUI/Sources/ViewControllerDescription/ViewControllerDescription.swift

```diff
@@ -70,7 +76,16 @@
         ///         `canUpdate(viewController:)` will result in an exception.
         ///
         /// - Parameter viewController: The view controller instance to update
```

Comment:
We may want to also add a note about the precondition being hit if the type is wrong here.

Reply:
```
Note: Passing a view controller that does not return `true` from
        ///         `canUpdate(viewController:)` will result in an exception.
```

Is in the note section, I can re-word that

Reply:
Done